dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -35,13 +35,13 @@
 from dragonchain import logger
 from dragonchain import exceptions
 from dragonchain.lib.interfaces import storage
+from dragonchain.lib.dto import eth, btc, bnb
 
 BROADCAST = os.environ["BROADCAST"]
 LEVEL = os.environ["LEVEL"]
 HTTP_REQUEST_TIMEOUT = 30  # seconds
 BROADCAST_RECEIPT_WAIT_TIME = 35  # seconds
-# TODO Make L5 wait time dynamic
-BROADCAST_RECEIPT_WAIT_TIME_L5 = 43200  # seconds
+L5_WAIT_TIMES: dict = {}  # initialize
```

Comment:
Even though this is module-level, we want to indicate this is for internal use only, so prepend the var with an underscore and lowercase it.
```suggestion
_l5_wait_times: Dict[str, int] = {}
```

Reply:
Fixed.